submissions/nallikaea/js-dom/index.js

```diff
@@ -0,0 +1,143 @@
+      'Вид базидіомікотових грибів родини Болетові. Їстівний смачний гриб.',
+    receptName:'Грибний суп з підосиковиків',
+    ingredients: 'Підосиковики 4шт, картопля - 3шт, цибуля 1шт, морква 1шт, сіль, перець чорний горошком, лавровий лист 2шт. ',
+    recept:'У каструлю влийте воду, посоліть, додайте гриби, спеції і варіть на середньому вогні 20 хвилин. Знімайте пінку шумівкою. Цибулю і моркву наріжте кубиками та обсмажте до золотистого кольору. Картоплю наріжте і додайте в грибний бульйон. Через кілька хвилин додайте смажені моркву і цибулю та варіть до готовності. Подавайте зі сметаною.'
+  }
+];
+
+const DOC = document;
+
+const container = DOC.createElement('div');
+container.classList.add('container');
+DOC.body.appendChild(container);
+
+const dictionaryName = DOC.createElement('h1');
+dictionaryName.classList.add('dictionary-name');
+dictionaryName.textContent = 'Мої знахідки';
+container.appendChild(dictionaryName);
+
+const divMain = DOC.createElement('div');
+divMain.classList.add('main');
+container.appendChild(divMain);
+
+const divMenu = DOC.createElement('div');
+divMenu.classList.add('menu');
+divMain.appendChild(divMenu);
+
+const navigation = DOC.createElement('ul');
+navigation.classList.add('navigation');
+const nav = DOC.querySelectorAll('li');
+divMenu.appendChild(navigation);
+
+gryby.forEach((val, key) => {
+  const li = DOC.createElement('li');
+  li.classList.add('list-element');
+  li.textContent = gryby[key].name;
+  if (key === 0) {
+    li.classList.add('clicked');
+  }
+  navigation.appendChild(li);
+});
+
+const divDescription = DOC.createElement('div');
+divDescription.classList.add('description');
+divMain.appendChild(divDescription);
+
+const divRecept = DOC.createElement('div');
+divRecept.classList.add('recept');
+divMain.appendChild(divRecept);
+
+const divImage = DOC.createElement('div');
+divImage.classList.add('image');
+divMain.appendChild(divImage);
+
+const descriptionHeader = DOC.createElement('h2');
+descriptionHeader.textContent = gryby[0].name;
+divDescription.appendChild(descriptionHeader);
+
+const descriptionParah = DOC.createElement('p');
+descriptionParah.textContent = gryby[0].description;
+divDescription.appendChild(descriptionParah);
+
+const descriptionReceptName = DOC.createElement('h2');
+descriptionReceptName.textContent = gryby[0].receptName;
+divDescription.appendChild(descriptionReceptName);
+
+const descriptionIng = DOC.createElement('h4');
+descriptionIng.textContent =  gryby[0].ingredients;
+divDescription.appendChild(descriptionIng);
+
+const descriptionRecept = DOC.createElement('p');
+descriptionRecept.textContent = gryby[0].recept;
+divDescription.appendChild(descriptionRecept);
+
+const image = DOC.createElement('img');
+image.classList.add('image');
+image.setAttribute('src', gryby[0].imgSrc);
+divImage.appendChild(image);
+
+const menuList = DOC.querySelector('ul');
+
+menuList.addEventListener('click', evt => {
+  const lis = DOC.querySelectorAll('li');
```

Comment:
You can just use `DOC.querySelector('.list-element.clicked').classList.remove('clicked')`
No need to get all of the elements and loop through them.